auth-web/src/services/keycloak.services.ts

```diff
@@ -46,6 +46,13 @@ class KeyCloakService {
     configHelper.addToSession(SessionStorageKeys.KeyCloakRefreshToken, this.kc.refreshToken)
     configHelper.addToSession(SessionStorageKeys.UserFullName, this.getUserInfo().fullName)
     this.parsedToken = this.kc.tokenParsed as UserToken
+
+    // Set flag in session storage so that common components will know account type
+    if (this.getUserInfo().userName.endsWith('@idir')) {
```

Comment:
There is a 'loginSource' claim in token which can be used here to identify the login type. Its value would be either BCSC or IDIR.